### PR TITLE
Kill hqadmin views

### PR DESCRIPTION
**Pull request by dannyroberts**
(only remaining one is deploy_history)

This contains https://github.com/dimagi/commcare-hq/pull/8384 and https://github.com/dimagi/commcare-hq/pull/9493

### CODE REVIEW

**Comment by czue** [comment]
this is gonna be a :tada: when it's done

**Comment by dannyroberts** [comment]
classic evening pact build error. Restarting build

**Comment by snopoke** [comment]
:+1: nice

File: corehq/apps/hqadmin/escheck.py [hunk 1]
```diff
@@ -7,6 +7,9 @@
 
 from casexml.apps.case.models import CommCareCase
 from corehq.apps.domain.models import Domain
+
+from corehq.apps.hqadmin.history import get_recent_changes
+from corehq.elastic import get_es
```
Comment (dimagimon) [inline corehq/apps/hqadmin/escheck.py]: F401 'get_es' imported but unused

**Comment by dimagimon** [comment]
There are 41 errors:

* corehq/apps/accounting/forms.py, line 198 - F841 local variable 'email_no_whitespace' is assigned to but never used
* corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py, line 2 - F401 'EntryPoint' imported but unused
* corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py, line 2 - F401 'SubscriptionType' imported but unused
* corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py, line 166 - E501 line too long (132 > 115 characters)
* corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py, line 172 - E501 line too long (116 > 115 characters)
* corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py, line 173 - E501 line too long (117 > 115 characters)
* corehq/apps/accounting/management/commands/cchq_software_plan_bootstrap.py, line 320 - W391 blank line at end of file
* corehq/apps/accounting/models.py, line 1205 - E124 closing bracket does not match visual indentation
* corehq/apps/accounting/subscription_changes.py, line 444 - E131 continuation line unaligned for hanging indent
* corehq/apps/api/resources/v0_5.py, line 82 - E126 continuation line over-indented for hanging indent
* corehq/apps/app_manager/models.py, line 5 - F401 'Mock' imported but unused
* corehq/apps/app_manager/models.py, line 20 - F401 'ResourceConflict' imported but unused
* corehq/apps/app_manager/models.py, line 30 - F401 'soft_assert' imported but unused
* corehq/apps/app_manager/models.py, line 3936 - E127 continuation line over-indented for visual indent
* corehq/apps/app_manager/models.py, line 3937 - E127 continuation line over-indented for visual indent
* corehq/apps/app_manager/models.py, line 3938 - E127 continuation line over-indented for visual indent
* corehq/apps/app_manager/models.py, line 3939 - E127 continuation line over-indented for visual indent
* corehq/apps/app_manager/tests/test_report_config.py, line 212 - E501 line too long (295 > 115 characters)
* corehq/apps/app_manager/views/releases.py, line 4 - F401 'method_decorator' imported but unused
* corehq/apps/app_manager/views/releases.py, line 19 - F401 'toggles' imported but unused
* corehq/apps/cloudcare/api.py, line 318 - W293 blank line contains whitespace
* corehq/apps/cloudcare/api.py, line 319 - W293 blank line contains whitespace
* corehq/apps/commtrack/management/commands/product_program_last_modified.py, line 7 - E302 expected 2 blank lines, found 1
* corehq/apps/custom_data_fields/management/commands/convert_custom_location_data.py, line 28 - E129 visually indented line with same indent as next logical line
* corehq/apps/custom_data_fields/management/commands/convert_custom_user_data.py, line 32 - E129 visually indented line with same indent as next logical line
* corehq/apps/domain/calculations.py, line 95 - E501 line too long (119 > 115 characters)
* corehq/apps/domain/calculations.py, line 113 - E501 line too long (119 > 115 characters)
* corehq/apps/domain/forms.py, line 958 - E202 whitespace before ')'
* corehq/apps/domain/forms.py, line 960 - E202 whitespace before ')'
* corehq/apps/cloudcare/static/cloudcare/js/util.js, line 132 - Missing semicolon.
* corehq/apps/commtrack/_design/filters/stock_reports.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/util.js, line 31 - Bad line breaking before '&&'.
* corehq/apps/commtrack/_design/util.js, line 31 - Missing semicolon.
* corehq/apps/commtrack/_design/views/domain_config/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/views/locations_by_code/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/views/product_by_code/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/views/product_by_program_id/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/views/products/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/_design/views/programs/map.js, line 1 - Missing name in function declaration.
* corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js, line 19 - Bad line breaking before '+'.
* corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js, line 20 - Bad line breaking before '+'.

**Comment by czue** [comment]
:guitar:

**Pull request merged**